src/main/java/com/ververica/flink/table/gateway/SqlCommandParser.java

```diff
@@ -242,6 +273,8 @@ private SqlCommandParser() {
 
 		DROP_VIEW,
 
+		DROP_VIEW_IF_EXISTS,
```

Comment:
do not add a new command, `DROP_VIEW_IF_EXISTS` is a special case of `DROP_VIEW`